lib/challenge.js

```diff
@@ -141,11 +150,18 @@ function prompt8() {
   // print someNumber to the console in your function so you can see its value
   console.log(someNumber);
 
-  // YOUR CODE HERE
+  var someNumber= Math.floor(Math.random()*100);
```

Comment:
⚠️ Please avoid using `var` moving forward! Best practice is to use `let` or `const`!